examples/montecarlo.jl

```diff
@@ -24,7 +24,8 @@ function montecarlo(mc_eval::Function, mc_monitor::Function, reps, n_returns, po
     end
 
     if mod(pernode,pooled) != 0
-        pooled = div(pernode, commsize-1)
+        pooled = Int64(div(pernode, commsize-1))
```

Comment:
Hi,
Looking at it again, I would say no, but I'm pretty sure that I added it
because in a usage case (a poorly chosen configuration) the argument was a
Float64. Perhaps I'm mistaken, though. At any rate, I don't think it will
do any harm.
Cheers,
M.

On Wed, Oct 21, 2015 at 2:53 PM, Andreas Noack <notifications@github.com>
wrote:

> In examples/montecarlo.jl
> <https://github.com/JuliaParallel/MPI.jl/pull/84#discussion_r42618363>:
>
> > @@ -24,7 +24,8 @@ function montecarlo(mc_eval::Function, mc_monitor::Function, reps, n_returns, po
> >      end
> >
> >      if mod(pernode,pooled) != 0
> > -        pooled = div(pernode, commsize-1)
> > +        pooled = Int64(div(pernode, commsize-1))
>
> Thanks. Is the Int64 necessary here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaParallel/MPI.jl/pull/84/files#r42618363>.
>